### DISCUSSION TITLE
How do I call api?

### DISCUSSION THREAD

**Discussion by Bhautik0110** [zitadel/zitadel#4781] [Q&A] [answered]
I want to call management API. Is that any way to create an organisation-wide token and set expiration like GitHub? Probably correct me If I'm wrong "currently the user's access token using we can call API". Thanks.

**Comment by hifabienne** [comment, marked as answer]
Hei @Bhautik0110 

You can use our service-users to call the management api. https://docs.zitadel.com/docs/guides/integrate/serviceusers
Then you have two options, either you use a key to authenticate your user with JWT Profile https://docs.zitadel.com/docs/guides/integrate/access-zitadel-apis

Or you can generate a PAT (Personal Access Token) and access the apis: https://docs.zitadel.com/docs/guides/integrate/pat

Make sure your service-user has the right permissions to access what you need.

**Reply by Bhautik0110** [reply]
Thank you @hifabienne :)
